stream-switch.js

```diff
@@ -149,6 +154,82 @@ Vue.component("stream-switch", {
             });
         },
 
+        loadManifest: function() {
+            var url = this.generateIngestUrl();
+            var vm = this;
+
+            this.dynamicMessage = this.loadingText;
+            this.showSpinner = true;
+            this.showSwitcherControls = false;
+            this.showVideoPlayer = false;
+
+            axios.get(url)
+                .then(function (response) {
+                    if (response === null || response.data === null || response.data.events === null) {
+                        return;
+                    }
+
+                    var hasAudioUrl = false;
+                    var hasVideoUrl = false;
+
+                    response.data.events.forEach(function (liveEvent) {
+                        if (!liveEvent.isLive) {
+                            return;
+                        }
+
+                        liveEvent.locators.forEach(function (locator) {
+                            if (liveEvent.name.toLowerCase() === vm.azureAudioChannelName.toLowerCase()) {
+                                hasAudioUrl = true;
+
+                                if (locator.type.toLowerCase() === "dash") {
+                                    vm.audioUrls.dash = locator.url;
+                                }
+
+                                if (locator.type.toLowerCase() === "hls") {
+                                    vm.audioUrls.hls = locator.url;
+                                }
+                            }
+
+                            if (liveEvent.name.toLowerCase() === vm.azureVideoChannelName.toLowerCase()) {
+                                hasVideoUrl = true;
+
+                                if (locator.type.toLowerCase() === "dash") {
+                                    vm.videoUrls.dash = locator.url;
+                                }
+
+                                if (locator.type.toLowerCase() === "hls") {
+                                    vm.videoUrls.hls = locator.url;
+                                }
+                            }
+                        });
+                    });
+
+                    vm.showSpinner = false;
+
+                    if (!hasAudioUrl && !hasVideoUrl) {
+                        vm.dynamicMessage = vm.offlineText;
+                        return;
+                    }
+
+                    vm.showSwitcherControls = response.data.isAllLive;
+                    vm.showVideoPlayer = response.data.isAnyLive;
+
+                    if (hasVideoUrl) {
+                        vm.selectedVideo();
+                    } else {
+                        vm.selectedAudio();
+                    }
+
+                    if (vm.timer != null) {
```

Comment:
Codacy found an issue: [Expected '!==' and instead saw '!='.](https://app.codacy.com/gh/literal-life-church/stream-switch/pullRequest?prid=5876146)